blocks/blockstore/blockstore.go

```diff
@@ -186,8 +186,9 @@ func (bs *blockstore) AllKeysChan(ctx context.Context) (<-chan *cid.Cid, error)
 		select {
 		case <-ctx.Done():
```

Comment:
since we're checking the context at the send layer (and this is in a goroutine, and not expected to really block) we can do away with the select and context check entirely here.